system/adb/Kconfig

```diff
@@ -0,0 +1,147 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig ADB_SERVER
+	bool "adb server command"
+	default n
```

Comment:
need depends or select kernel adb driver, or adbd can work without usb but tcp?

Reply:
The idea is once the server is enabled we can select at least one of USB or TCP transports. Or both if needed.
ADBD_USB must indeed depend on kernel adb driver.